.changeset/spotty-mirrors-care.md

```diff
@@ -0,0 +1,11 @@
+---
+"@comet/dev-process-manager": minor
+---
+
+Watch dev-pm.config.js and update in running daemon
+
+This allows modifications without having to shutdown the deamon.
+- processes are not restarted if the script command changes
```

Comment:
Is this the desired behavior? I think a restart would be more logical

Reply:
I don't think I would expect that. And you would also have edge-cases like waitOn or env get modified - should that also trigger a restart?

Reply:
I've thought mostly about waitOn end script. Maybe we could change the log to something like "... updated, please restart to apply changes"

Reply:
Okay, you've convinced me, let's keep it like this 😁